dracut/anaconda-lib.sh

```diff
@@ -117,14 +117,15 @@ anaconda_net_root() {
         info "anaconda: successfully fetched stage2 from $repo"
 
         # NOTE: Should be the same as anaconda_auto_updates()
-        updates=$(fetch_url $repo/images/updates.img 2> /tmp/updates_err)
-        [ -z "$updates" ] && debug_msg $(cat /tmp/updates_err)
-        [ -n "$updates" ] && unpack_updates_img $updates /updates
-
-        product=$(fetch_url $repo/images/product.img 2> /tmp/product_err)
-        [ -z "$product" ] && debug_msg $(cat /tmp/product_err)
-        [ -n "$product" ] && unpack_updates_img $product /updates
-
+        if ! getargbool 0 updates inst.updates; then
+            updates=$(fetch_url $repo/images/updates.img 2> /tmp/updates_err)
+            [ -z "$updates" ] && debug_msg $(cat /tmp/updates_err)
+            [ -n "$updates" ] && unpack_updates_img $updates /updates
+
+            product=$(fetch_url $repo/images/product.img 2> /tmp/product_err)
+            [ -z "$product" ] && debug_msg $(cat /tmp/product_err)
+            [ -n "$product" ] && unpack_updates_img $product /updates
```

Comment:
I don't think updates images should be skipped based on ``inst.updates`` kernel param.

Their are basically the same but product image is specialized on rebranding so it shouldn't be excluded based on updates_image.